python/pyspark/sql/functions.py

```diff
@@ -1142,13 +1384,23 @@ def sinh(col: "ColumnOrName") -> Column:
     Parameters
     ----------
     col : :class:`~pyspark.sql.Column` or str
-        hyperbolic angle
+        hyperbolic angle in radians
```

Comment:
I dont' believe hyperbolic angles are measured in radians

Reply:
my bad, reverted